packages/react-button/src/components/Button/Button.test.tsx

```diff
@@ -1,14 +1,93 @@
 import * as React from 'react';
 import * as path from 'path';
+import * as ReactTestUtils from 'react-dom/test-utils';
+import * as renderer from 'react-test-renderer';
+import { mount, ReactWrapper } from 'enzyme';
+import { EnterKey, SpacebarKey } from '@fluentui/keyboard-key';
 import { isConformant } from '@fluentui/react-conformance';
 import { Button } from './Button';
-import * as renderer from 'react-test-renderer';
 import { ButtonRef } from './Button.types';
-import { mount, ReactWrapper } from 'enzyme';
 
-describe('Button', () => {
-  let wrapper: ReactWrapper | undefined;
+let wrapper: ReactWrapper | undefined;
+
+describe('Button behavior', () => {
+  afterEach(() => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = undefined;
+    }
+  });
+
+  it('renders Button with the correct default accessibility props', () => {
+    const onClick = jest.fn();
+
+    wrapper = mount(<Button onClick={onClick}>Button</Button>);
+
+    const button = wrapper.find('button');
+
+    expect(button.getElements().length).toBe(1);
+    expect(button.props().role).toBe(undefined);
+    expect(button.props().tabIndex).toBe(undefined);
```

Comment:
```suggestion
    expect(button.prop('role')).toBe(undefined);
    expect(button.prop('tabIndex')).toBe(undefined);
```

_nit:_ Enzyme has support of `.prop()`

Reply:
Went with your suggestion.